sdk/go/pulumi-language-go/main.go

```diff
@@ -114,10 +122,14 @@ func main() {
 		cmdutil.Exit(errors.Wrapf(err, "could not start health check host RPC server"))
 	}
 
+	if binary != "" && buildTarget != "" {
+		cmdutil.Exit(errors.Errorf("binary and buildTarget cannot both be specified"))
```

Comment:
Need a bit of a reference here where the user should fix it, it's in Pulumi.yaml right? Consider a confused user reading this msg.

Reply:
```suggestion
		cmdutil.Exit(errors.Errorf("cannot provide both `binary` and `buildTarget` in Pulumi.yaml"))
```